lib/site_mercado/resources/v1/order.rb

```diff
@@ -25,7 +25,7 @@ def separated(code, separator:, amount_cold_bag: 0, amount_dry_bag: 0, total_val
       end
 
       def cancel(code, message: nil)
-        params = { messagem: message }
+        params = { mensagem: message }
         Client.put("/pedido/#{code}/status/cancelar", body: params)
```

Comment:
:nail_care: :warning: 

Nesses casos onde a var é usada uma unica vez e cabe inlin, eu tenho preferido passar assim:

```suggestion
         Client.put("/pedido/#{code}/status/cancelar", body: { mensagem: message })
```